packages/doenetml-worker-rust/lib-rust-macros/src/component_module/items/props.rs

```diff
@@ -169,12 +199,31 @@ impl PropsEnum {
                 descriptions.push("- Private: this prop can only be used internally.".to_string())
             }
         }
-        match variant.for_render {
-            true => descriptions.push(
-                "- ForRender: this prop is always rendered and available to the UI.".to_string(),
-            ),
-            false => descriptions.push("- NotForRender: this prop is not rendered.".to_string()),
+
+        #[allow(clippy::collapsible_else_if)]
+        if variant.for_render().in_graph {
```

Comment:
Since this is exhaustive, it could be done with a `match (variant.for_render().in_graph, variant.for_render().in_text) { ... }`